cantools/subparsers/generate_c_source.py

```diff
@@ -118,4 +119,8 @@ def add_subparser(subparsers):
     generate_c_source_parser.add_argument(
         '--node',
         help='Generate pack/unpack functions only for messages sent/received by the node.')
+    generate_c_source_parser.add_argument(
```

Comment:
as previously mentioned: I would like to generate these functions unconditionally..

Reply:
Okay, I'll remove additional argument